ui/src/react-admin/modules/content-page/components/blocks/BlockMaintainersGrid/BlockMaintainersGrid.tsx

```diff
@@ -0,0 +1,69 @@
+import classnames from 'classnames';
+import { FunctionComponent, ReactElement } from 'react';
+import { HeadingTypeOption } from '~modules/content-page/types/content-block.types';
+import { DefaultComponentProps } from '~modules/shared/types/components';
+import { BlockHeading } from '../BlockHeading';
+import { ButtonAction } from '@viaa/avo2-components';
+import SmartLink from '~modules/shared/components/SmartLink/SmartLink';
+
+export interface BlockMaintainerGridProps extends DefaultComponentProps {
+	title: string;
+	titleType: HeadingTypeOption;
+	subtitle: string;
+	buttonLabel: string;
+	buttonAction?: ButtonAction;
+	maintainers: { imageSrc: string; linkAction?: ButtonAction }[];
+}
+
+export const BlockMaintainersGrid: FunctionComponent<BlockMaintainerGridProps> = ({
+	className,
+	title,
+	titleType,
+	subtitle,
+	buttonLabel,
+	buttonAction,
+	maintainers,
+}): ReactElement => {
+	return (
+		<article className={classnames('c-block-maintainers-grid', className)}>
+			<div className="c-block-maintainers-grid__header">
+				<div>
+					<BlockHeading className="c-block-maintainers-grid__title" type={titleType}>
+						{title}
+					</BlockHeading>
+					{subtitle && <p>{subtitle}</p>}
+				</div>
+				{buttonLabel && (
+					<div>
+						<SmartLink action={buttonAction}>{buttonLabel}</SmartLink>
+					</div>
+				)}
+			</div>
+			<ul>
+				{maintainers
+					.filter((m) => !!m.imageSrc)
+					.map((maintainer, index) => {
+						return (
+							<li
+								key={
+									maintainer.linkAction?.value +
+									'--' +
```

Comment:
![image](https://user-images.githubusercontent.com/113892598/235932249-77e88dc7-8112-4a96-bb20-10466f77d2ba.png)
use SEPARATOR

Reply:
I changed it to 
```
key={`${maintainer.linkAction?.value}-${maintainer.imageSrc}-${index}`}
```

Since we never want to split on that value, i don't think a const is needed. We just want to separate the values.
The rest of the codebase also uses a single quotes, so i also changed that.